Gemfile.lock

```diff
@@ -27,47 +27,31 @@ PATH
 GEM
```

Comment:
I think that it's a case of it depends, if you have something that is more an application than a library, then perhaps you should. Certainly I think this plugin with it's dependencies is in a grey area.